### ISSUE TITLE
Random shift appears between groundtruth and rendered images with nerfstudio-data format and Nerfacto method

### ISSUE THREAD

**Issue by ashutoshmishra1014** [nerfstudio-project/nerfstudio#1291]
**Describe the bug**
The rendered outputs(rgb+depth) seem to be randomly shifted when using Nerfacto/Depth-Nerfacto with Nerfstudio dataparser.

**To Reproduce**
I used the below command for training
1. ns-train nerfacto --data \<path-to-dataset\> --vis tensorboard --viewer.quit-on-train-completion True --pipeline.datamanager.eval_num_images_to_sample_from 50 --steps_per_eval_all_images 35000

**Expected behavior**
The ground truth rgb image and rendered rgb/depth image should be pixel aligned.

**Screenshots**
![chair](https://user-images.githubusercontent.com/24511198/214808005-9f697eaf-340a-44a5-b01e-32d121fa89e4.png)
![sofa](https://user-images.githubusercontent.com/24511198/214812088-180c7e7b-06a7-4a28-89e1-11affdd52cde.png)
The above results are for the redwood3dscan dataset.
The left image is the ground truth rgb, the center one is rendered rgb and the last one is the rendered depth image.
Have added a straight black line to show the issue of displacement.

**Additional context**
I have observed same issue with multiple other real datasets.
Process of preparing redwood3dscan dataset:
1. run camera calibration with COLMAP
2. converted COLMAP camera parameters to NeRF format(transforms.json) using CAMORPH(https://github.com/Fraunhofer-IIS/camorph)
3. use the below command for training:
4. ns-train nerfacto --data <path-to-dataset> --vis tensorboard --viewer.quit-on-train-completion True --pipeline.datamanager.eval_num_images_to_sample_from 50 --steps_per_eval_all_images 35000

Note:
_I have faced same issue with a polycam captured dataset and preparing it using "ns-process-data polycam ........" and then running the training command._

**Comment by HrsPythonix** [comment]
Maybe this could be related to pose refinement in nerfacto? 

https://docs.nerf.studio/en/latest/nerfology/methods/nerfacto.html

**Comment by tancik** [comment]
Yes, @HrsPythonix is correct, if you want, you can turn off the pose optimization with `--pipeline.datamanager.camera-optimizer.mode off`

@ethanweber is also working on a PR that add pose optimization to the eval images in #1139

**Comment by ashutoshmishra1014** [comment]
@tancik @HrsPythonix 
I suspected the same so I have already tried couple of runs with --pipeline.datamanager.camera-optimizer.mode off but still the same issue.

**Comment by tancik** [comment]
Can you share your processed data.

**Comment by ashutoshmishra1014** [comment]
@tancik I realised the issue was on my part. 
For my dataset, I created 3 separate splits for train, validation and test.
Nerfacto dataparser performs preprocessing such as orientation, centering and scaling using all the poses from the dataset and then creates the training and validation split.
In my implementation, the preprocessing was done on the individual split of data which of course did not match the above behaviour and hence the random shift.

This leads me to the next short question.
How to control the preprocessing, orientation_method etc. in the training command?

The command
ns-train nerfacto -h 
does not let me specify these nerfacto specific arguments.
